test/mpi/datatype/hvecblklen.c

```diff
@@ -14,7 +14,7 @@
 */
 int main(int argc, char *argv[])
 {
-    MPI_Datatype ot, ot2, newtype;
+    MPI_Datatype dt, dt2, newtype;
```

Comment:
I don't know the original motivation for using `ot`, but I'm not sure it qualifies as a spelling mistake.